_i18n/pl/keyboards.md

```diff
@@ -0,0 +1,1225 @@
+---
+title: "Wszystkie wspierane klawiatury"
+lang: "pl"
+layout: qmk-editable
+permalink: /pl/keyboards/
+# this page is automatically generated
+---
+## Wszystkie obsługiwane klawiatury
+
+Ta strona jest generowana automatycznie z klawiatur znajdujących się obecnie w QMK. Domyślne pliki .hex są aktualne z najnowszą wersją QMK i są wypychane przy każdej kompilacji za pośrednictwem Travisa.
+
+Chcesz stronę qmk.fm? [Utwórz dla niego plik `<keyboard> .md`] (https://github.com/qmk/qmk.fm/tree/gh-pages/_pages/keyboards)! (użyj innych jako przykładów / szablonów)
+
+Obecnie dostępne są tylko pliki .hex - niektóre z wymienionych tutaj mogą nie być tym, co jest potrzebne do flashowania klawiatury. W takim przypadku [proszę otworzyć problem] (https://github.com/qmk/qmk.fm/issues)!
```

Comment:
This statement is from the English page, but is no longer true – `.bin` files are provided when appropriate.